packages/psss/src/api/queries/useSingleWeeklyReport.js

```diff
@@ -0,0 +1,130 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { queryCache } from 'react-query';
+import { useData } from './useData';
+
+const getAlerts = row =>
+  Object.entries(row).reduce(
+    (list, [key, value]) =>
+      key.includes('Threshold Crossed') && value
+        ? [...list, key.split('Threshold Crossed')[0]]
```

Comment:
To avoid trailing spaces:

```suggestion
        ? [...list, key.split(' Threshold Crossed')[0]]
```